src/bindings/html/head.js

```diff
@@ -3,7 +3,7 @@
 export function getResourceLinks(head) {
   return Array.prototype.map.call(
     head.querySelectorAll('link[rel="localization"]'),
-    (el) => el.getAttribute('href'));
+    el => decodeURI(el.getAttribute('href')));
```

Comment:
Some apps have own build scripts which use `nsIDOMSerializer` to serialize the HTML, which escapes `{}` in `href` attributes.  We ran into this in the Keyboard app in https://bugzil.la/1098188.